src/main/java/net/fabricmc/loader/launch/knot/KnotClassDelegate.java

```diff
@@ -62,6 +62,7 @@
 	private final boolean isDevelopment;
 	private final EnvType envType;
 	private MixinTransformer mixinTransformer;
+	private volatile boolean isLoadingMixinTransformer;
```

Comment:
Does this need to be volatile?

Reply:
I'm not sure - I don't fully understand how classloading works but as far as I know it's at least meant to be able to be called from multiple threads at once? That being said seeing as `mixinTransformer` *isn't* volatile there's probably no reason for `isLoadingMixinTransformer` to be volatile either.